sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -433,6 +438,78 @@ public EditCaseSteps(
+          List<XWPFParagraph> paragraphList = xdoc.getParagraphs();
+          String[] line = paragraphList.get(29).getText().split(":");
+          softly.assertEquals(line[0], "Report Date");
+          line = paragraphList.get(31).getText().split(":");
+          softly.assertEquals(line[0], "Vaccination Date");
+          softly.assertEquals(
+              line[1].trim(),
+              CreateNewVaccinationSteps.vaccination
+                  .getVaccinationDate()
+                  .format(DateTimeFormatter.ofPattern("dd.MM.yyyy")));
+          line = paragraphList.get(32).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine name");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getVaccineName());
+          line = paragraphList.get(33).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine name Details");
+          line = paragraphList.get(34).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine Manufacturer");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getVaccineManufacturer());
+          line = paragraphList.get(35).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine Manufacturer details");
+          line = paragraphList.get(36).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine Type");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getVaccineType());
+          line = paragraphList.get(37).getText().split(":");
+          softly.assertEquals(line[0], "Vaccine Dose");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getVaccineDose());
+          line = paragraphList.get(38).getText().split(":");
+          softly.assertEquals(line[0], "INN");
+          softly.assertEquals(line[1].trim(), CreateNewVaccinationSteps.vaccination.getInn());
+          line = paragraphList.get(39).getText().split(":");
+          softly.assertEquals(line[0], "Batch");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getBatchNumber());
+          line = paragraphList.get(40).getText().split(":");
+          softly.assertEquals(line[0], "UNII Code");
+          softly.assertEquals(line[1].trim(), CreateNewVaccinationSteps.vaccination.getUniiCode());
+          line = paragraphList.get(41).getText().split(":");
+          softly.assertEquals(line[0], "ATC Code");
+          softly.assertEquals(line[1].trim(), CreateNewVaccinationSteps.vaccination.getAtcCode());
+          line = paragraphList.get(42).getText().split(":");
+          softly.assertEquals(line[0], "Vaccination Info Source");
+          softly.assertEquals(
+              line[1].trim(), CreateNewVaccinationSteps.vaccination.getVaccinationInfoSource());
+          softly.assertAll();
```

Comment:
add assert messages for all checks, otherwise we won't be able to identify the issue from report